webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlSerialisationImpl.scala

```diff
@@ -197,7 +198,7 @@ class DirectiveSerialisationImpl(xmlVersion:String) extends DirectiveSerialisati
     , directive          : Directive
   ) = {
     createTrimedElem(XML_TAG_DIRECTIVE, xmlVersion) (
-          <id>{directive.id.value}</id>
+          <id>{directive.id.uid.value}</id>
```

Comment:
Shouldn't we store the revision here ?

Reply:
no, by definition, it will be given by the git commit